src/components/visualizations/typings/legend.ts

```diff
@@ -9,6 +9,7 @@ export interface ILegendOptions {
     position: PositionType;
     format: string;
     items: LegendOptionsItemType[];
+    responsive?: boolean;
```

Comment:
do we really need this?

Reply:
Yes, we do since I have refactored this [line](https://github.com/gooddata/gooddata-react-components/pull/1248/files#diff-5ae23ede9709e91ce9df7ca2071559a8R39)